sdk/resourcehealth/Azure.ResourceManager.ResourceHealth/api/Azure.ResourceManager.ResourceHealth.netstandard2.0.cs

```diff
@@ -425,18 +521,11 @@ public partial class EventPropertiesRecommendedActionsItem
         public static bool operator !=(Azure.ResourceManager.ResourceHealth.Models.EventTypeValue left, Azure.ResourceManager.ResourceHealth.Models.EventTypeValue right) { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class Faq
-    {
-        internal Faq() { }
-        public string Answer { get { throw null; } }
-        public string LocaleCode { get { throw null; } }
-        public string Question { get { throw null; } }
-    }
-    public partial class Impact
+    public partial class EventUpdate
```

Comment:
well these models that start with `Event` look like to have a generic name, please consider to change them

Reply:
fixed